src/content/en/fundamentals/media/fast-playback-with-video-preload.md

```diff
@@ -0,0 +1,559 @@
+parsed
+      </td>
+    </tr>
+    <tr>
+      <td>
+Doesn't work with MSE
+      </td>
+    </tr>
+    <tr>
+      <td rowspan=3 style="white-space: nowrap">
+<a href="#link_preload">Link preload</a>
+      </td>
+      <td>
+Forces the browser to make a request for a video resource without blocking
+the document's <code>onload</code> event
+      </td>
+      <td rowspan=3>
+Doesn't work with HTTP Range requests
+      </td>
+    <tr>
+      <td>
+Compatible with MSE and segment files
+      </td>
+    </tr>
+    <tr>
+      <td>
+Should be used only for audio files or short video clips when fetching full
+resource
+      </td>
+    </tr>
+    <tr>
+      <td>
+<a href="#manual_buffering">Manual buffering</a>
+      </td>
+      <td>
+Full control
+      </td>
+      <td>
+Complex error handling is website responsibility
+      </td>
+    </tr>
+  </tbody>
+</table>
+
+## Video preload attribute
+
+If the video source is an unique file hosted on a web server, you may want to
+use the video `preload` attribute to provide a hint to the browser as to [how
+much information or content to preload]. This means [Media Source Extensions
+(MSE)] is not compatible with `preload`.
+
+Resource fetching will start only when initial HTML document has been
+completely loaded and parsed (eg. `DOMContentLoaded` event) while the very
+different `load` event will be fired when resource has actually been fetched.
+
+<figure class="TODO">
+  <img src="/web/fundamentals/media/images/video-preload/video-preload.svg">
+</figure>
+
+Setting `preload` value to `metadata` indicates that the user is not expected
+to need the video, but that fetching its metadata (dimensions, track list,
+duration, and so on) is desirable.
+
+```
+<video id="video" preload="metadata" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('progress', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+With `preload` value set to `auto`, we're saying th at downloading the entire
+video right away is considered desirable.
+
+```
+<video id="video" preload="auto" src="file.mp4" controls></video>
+
+<script>
+  video.addEventListener('progress', function() {
+    if (video.buffered.length === 0) return;
+
+    var bufferedSeconds = video.buffered.end(0) - video.buffered.start(0);
+    console.log(bufferedSeconds + ' seconds of video are ready to play!');
+  });
+</script>
+```
+
+There are some caveats though. As this is just a hint, browser may completely
+ignore the `preload` attribute. At the time of writing, here are some rules
+applied in Chrome:
+
+- When [Data Saver] is enabled, Chrome forces `preload` value to `none`.
+- In Android 4.3, Chrome forces `preload` value to `none`.
+- On a cellular connection (2G, 3G, and 4G), Chrome forces `preload` value to
+  `metadata`.
+
```

Comment:
Can you add a section on best practices? E.g. Prefer preload=metadata or even preload=none+poster if you have lots of resources on the same domain to avoid hitting the 6 connection HTTP connection cap which can hang loading of resources. This may also improve page speed if video is a secondary attraction on the page; i.e. you don't want to slow down page loading and scrolling by loading in clips that are unlikely to be used or where a brief delay is fine. In these cases prefer preload=none+poster.